### DISCUSSION TITLE
Best practice for segment_duration & partition_number

### DISCUSSION THREAD

**Discussion by zouxiang1993** [CeresDB/ceresdb#768] [Q&A] [answered]
I think these settings are critical for performance, but I don't know how to choose a proper value.

**Comment by ShiKaiWi** [comment, marked as answer]
Sorry for late response, we didn't pay enough attention to the discussions here. As for `segment duration` tuning, I would say let the ceresdb decide it for you (the `segment duration` is decided by sampling first written data). And `partition number` can be decided easily for now, I guess a proper partition number should be 3-4 times physical nodes for expansion in the future.
